examples/qinstall-config.yml

```diff
@@ -0,0 +1,264 @@
+---
+# yaml-language-server: $schema=config.schema.json
+#
+# name of the cluster
+project_name: az-hop
+# azure location name as returned by the command : az account list-locations -o table
+location: westus2
+# Name of the resource group to create all resources
+resource_group: azhop-qinstall
+# If using an existing resource group set to true. Default is false
+# When using an existing resource group make sure the location match the one of the existing resource group
+use_existing_rg: false
+# If set to true, will disable telemetry for azhop. See https://azure.github.io/az-hop/deploy/telemetry.html.
+#optout_telemetry: true
+
+tags:
+  env: dev
+  project: azhop-quick-install
+# Define an Azure Netapp Files (ANF) account, single pool and volume
+# If not present, assume that there is an existing NFS share for the users home directory
+anf:
+  create: true
+  # Size of the ANF pool and unique volume (min: 4TB, max: 100TB)
+  homefs_size_tb: 4
+  # Service level of the ANF volume, can be: Standard, Premium, Ultra
+  homefs_service_level: Standard
+  # dual protocol
+  dual_protocol: false # true to enable SMB support. false by default
+  # If alerting is enabled, this value will be used to determine when to trigger alerts 
+  alert_threshold: 80 # alert when ANF volume reaches this threshold
+
+# For small deployments you can use Azure Files instead of ANF for the home directory
+azurefiles:
+  create: false
+  size_gb: 1024
+
+# These mounts will be listed in the Files menu of the OnDemand portal and automatically mounted on all compute nodes and remote desktop nodes
+mounts:
+  # mount settings for the user home directory
+  home: # This home name can't be changed
+    type: anf # anf or azurefiles, default to anf. One of the two should be defined in order to mount the home directory
+    mountpoint: /anfhome # /sharedhome for example
+    server: '{{anf_home_ip}}' # Specify an existing NFS server name or IP, when using the ANF built in use '{{anf_home_ip}}'
+    export: '{{anf_home_path}}' # Specify an existing NFS export directory, when using the ANF built in use '{{anf_home_path}}'
+    options: '{{anf_home_opts}}' # Specify the mount options. Default to rw,hard,rsize=262144,wsize=262144,vers=3,tcp,_netdev
+
+# name of the admin account
+admin_user: hpcadmin
+# List of identities (object ids) to grant read access to az-hop key vault (optional)
+# key_vault_readers:
+# Network
+network:
```

Comment:
to simplify the network section, remove create_nsg, then all the name and the create properties the subnets

Reply:
Removing those lines as well.